packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -138,8 +138,8 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
       allowDisabledFocus: true,
       role: 'menuitem',
       ...item,
-      styles: { root: { height: '100%' }, label: { whiteSpace: 'nowrap' }, ...item.buttonStyles },
-      className: css('ms-CommandBarItem-link', item.className),
+      styles: { label: { whiteSpace: 'nowrap' }, ...item.buttonStyles },
+      className: css(this._classNames.commandBarItems, item.className),
```

Comment:
what about specificity? these will both be the same specificity. if user provides a height, can we ommit the className?

Reply:
@micahgodbolt, you're right, thanks for that, I didn't really catch it! I'm wondering if there is a cleaner way to do this. I'll investigate.